st2auth/conf/apache.full.conf

```diff
@@ -0,0 +1,49 @@
+<VirtualHost *:443>
+
+    ServerName 172.168.50.50:443
```

Comment:
This apache conf needs to be inside vagrant, right? Why can't we use loopback IP here?